07-Customizing-AzSK-for-your-Org/Extending AzSK Module/Readme.md

```diff
@@ -269,65 +272,64 @@ class SubscriptionCore: SVTBase
    Refer the below screenshot.  
    ![Block diagram of AzSK extension](../../Images/08_AzSK_Extension_SVT_Storage_Policy_Example.png)
   
-6. 	 That's it!! You can now scan the new extended control like any other control.
+6. 	 The new control is now added the SubscriptionCore SVT. The new extended control can be scanned like any other control.
   
 ```PowerShell
 	Get-AzSKSubscriptionSecurityStatus -SubscriptionId '<sid>' -ControlIds 'Azure_Subscription_AuthZ_Limit_Admin_Count_Ext'
 ```
 
 ##### B. Extending a GRS SVT
 
-We will add a mock control that 'storage accounts must be created in 'eastus2' region.
+Below is an example to add a mock control that 'storage accounts must be created in 'eastus2' region.
```

Comment:
There's a mismatch of quotes - There's no closing ' for **'**storage accounts must...

Reply:
Done